src/shared/components/data-table/components/table-controls/components/pagination/Pagination.tsx

```diff
@@ -22,18 +22,21 @@ import { TableContext } from 'src/shared/components/data-table/DataTable';
 import styles from './Pagination.scss';
 
 const Pagination = () => {
-  const { dispatch, currentPageNumber, data, rowsPerPage } = useContext(
-    TableContext
-  ) || {
-    currentPageNumber: 1,
-    rowsPerPage: 100
-  };
+  const { dispatch, currentPageNumber, data, rowsPerPage, hiddenRowIds } =
+    useContext(TableContext) || {
+      currentPageNumber: 1,
+      rowsPerPage: 100,
+      hiddenRowIds: {}
+    };
 
   if (!dispatch || !data) {
     return null;
   }
 
-  const highestPageNumber = Math.ceil(data?.length / rowsPerPage);
+  const totalHiddenRows = Object.keys(hiddenRowIds).length;
```

Comment:
Since it's `.length`, the name should probably say count? At the moment, it may mean the rows themselves.

Reply:
That's done